plugins/scaffolder-backend/api-report.md

```diff
@@ -428,8 +428,18 @@ export class DatabaseTaskStore implements TaskStore {
   // (undocumented)
   heartbeatTask(taskId: string): Promise<void>;
   // (undocumented)
-  list(options: { createdBy?: string; status?: TaskStatus_2 }): Promise<{
+  list(options?: {
+    createdBy?: string | string[];
+    status?: TaskStatus_2 | TaskStatus_2[];
+    limit?: number;
+    offset?: number;
+    order?: {
+      order: 'asc' | 'desc';
+      field: string;
+    }[];
+  }): Promise<{
     tasks: SerializedTask_2[];
+    totalTasks: number;
```

Comment:
I think we need to make this optional in the API too, otherwise again, it's breaking for implementers of the API.